web/src/pages/logs-dev-page.tsx

```diff
@@ -31,6 +31,8 @@ t('plugin__logging-view-plugin~Logs')
 const LogsDevPage: React.FunctionComponent = () => {
   const { ns: namespace } = useParams<{ ns: string }>();
   const [isHistogramVisible, setIsHistogramVisible] = React.useState(false);
+  const initialTenant = getInitialTenantFromNamespace(namespace);
+  const tenant = React.useRef(initialTenant);
```

Comment:
```suggestion
  const tenant = getInitialTenantFromNamespace(namespace);
```
I think this is the same case as a comment I had on a previous PR: https://github.com/openshift/logging-view-plugin/pull/52#pullrequestreview-1237157757

I think `useRef` doesn't bring any benefit in this case.